mucommander-viewer-image/src/main/java/com/mucommander/viewer/image/ImageViewer.java

```diff
@@ -86,7 +95,7 @@ class ImageViewer implements FileViewer, ActionListener {
     private JScrollPane scrollPane = new JScrollPane();
     private ImageStatusPanel statusPanel = new ImageStatusPanel();
 
-    private InitialZoom initialZoom = InitialZoom.RESIZE_WINDOW;
+    private InitialZoom initialZoom = InitialZoom.FIT_TO_WINDOW;
```

Comment:
what's the initial size of the window?

Reply:
I will check

Reply:
I think it is default size for JPanel (but not 100% sure)